### DISCUSSION TITLE
Text truncation depending on number of words/symbols

### DISCUSSION THREAD

**Discussion by dubrovenskaya** [AvaloniaUI/Avalonia#8923] [Q&A]
Hello together, 
I have accidentally encountered a problem that I have not seen before. Namely, the texts are cut off differently depending on the number of words or symbols in the text, so that sometimes no text is visible, although the text itself is there. In the picture, you can see that the text property has the value "Do you want to save the changes?" The text is not fully displayed. But if I add additional spaces (3 symbols) in DevTool - the text is displayed completely. Does anyone have any idea how I could solve the problem? Thanks in advance!
![MessageBox1](https://user-images.githubusercontent.com/33326526/189341323-af2dcf73-8537-4c52-9066-c20f56501a9e.png)
![MessageBox2](https://user-images.githubusercontent.com/33326526/189341586-a675af05-23fb-4127-a334-ceeeb1f74e37.png)
![MessageBox3](https://user-images.githubusercontent.com/33326526/189341348-175a5588-493f-474d-8754-7228d58f632a.png)
![MessageBox4](https://user-images.githubusercontent.com/33326526/189341358-7470d331-118c-43ce-be22-2d8162812d80.png)

**Comment by jp2masa** [comment]
I think this is the same as #8838.
